Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs

```diff
@@ -489,7 +489,7 @@ public enum TouchPhase
     /// </remarks>
     [InputControlLayout(stateType = typeof(TouchscreenState), isGenericTypeOfDevice = true)]
     [Scripting.Preserve]
-    public class Touchscreen : Pointer, IInputStateCallbackReceiver
+    public class Touchscreen : Pointer, IInputStateCallbackReceiver, IEventMerger
```

Comment:
Probably needs an exclusion to pass the "no API breakage" test.

Reply:
Inheriting from internal class breaks api? oof ...

Reply:
fixed